pycsghub/cli.py

```diff
@@ -0,0 +1,56 @@
+import typer
+from typing import Annotated, List, Optional
+from pycsghub.cmd import repo
+from pycsghub.cmd.repo_types import RepoType
+from pycsghub.constants import DEFAULT_CSGHUB_DOMAIN, DEFAULT_REVISION
+
+app = typer.Typer(add_completion=False)
+
+OPTIONS = {
+    "repoType": typer.Option(default=..., help="Specify the repository type."),
+    "repoID": typer.Option(default=..., help="Specify the repository ID."),
+    "repoFiles": typer.Option(default=..., help="Local path to the file or files to upload. Defaults to the relative path of the file of repo of OpenCSG Hub."),
+    "revision": typer.Option(default=..., help="An optional Git revision id which can be a branch name"),
+    "cache_dir": typer.Option(default=..., help="Path to the directory where to save the downloaded files."),
+    "endpoint": typer.Option(default=..., help="The address of the request to be sent."),
+    "token": typer.Option(default=..., help="A User Access Token generated from https://opencsg.com/settings/access-token"),
```

Comment:
please support reading token from `env`

Reply:
those are optional, and get from env by default.